tests/bell.py

```diff
@@ -90,7 +93,129 @@ def logout(driver):
     # wait 5 secs
     sleep(5)
 
+def goto(driver, page):
+    """ string -> NoneType
+
+    Goes to the provided page (use "Home" to go to the homepage)
+    """
+    if page == "Home":
+        driver.get(get_url())
+        sleep(5)
+        return
+    elif driver.current_url == get_url():
+        login(driver, "admin", "password")
+    link = driver.find_element_by_link_text(page)
+    link.click()
+    sleep(5)
+
+def add_member(driver, first_name = "", last_name = "", middle_name = "",
+               login = "", password = "", phone = "", email = "", language = "",
+               birthdate = None, gender = "", level = ""):
+    """ object, str, str, str, str, str, str, str, str, 
+    list[str, str, str], str, str -> tuple(str, str)
+
+    Goes to homepage, fills in new member form, and returns a tuple of strings
+    representing username and password of the new member or, if 
+    the member already exists, return a tuple of empty strings.
+    TODO: Needs testing AND refactoring!
+    """
+    birthdate = birthdate or []
```

Comment:
When before I added that [QuantifiedCode was whining about the arguments ](https://www.quantifiedcode.com/app/project/gh:ole-vi:BeLL-Apps/diff/879bacddb848c4fbf198769ff10e004da34db40a/7e47ebef72da3bc4f01de68ba47eb6dc36f01889/issues?groups=code_patterns%3A3P0qV6OB%3Af0) so I was lazy and just took their solution from [here.](https://www.quantifiedcode.com/app/issue_class/3P0qV6OB)

I'm sure your solution would work fine too, and is probably more clear, I just wanted to get rid of the warning so we could continue with the rest of the meeting 😋